addons/web/static/src/views/calendar/calendar_renderer.scss

```
@@ -205,10 +205,12 @@
                 }
 
                 &.fc-today {
-                    color: $primary;
+                    --o-cw-color: #fff;
```

To be tested, but I find safer to do compute the color  (other occurrences as well):
```suggestion
                    --o-cw-color: #{color-contrast($o-cw-color-today-accent)};
```